README.md

```diff
@@ -1,3 +1,114 @@
+
+## Demo Application Setup ##
+
+**Step 1:** Connect your computer to the LAN of your Ewon device, and open an FTP connection to your Ewon device. To create an FTP connection, using your file manager (i.e. Windows Explorer), enter `ftp://[EWON IP HERE]/` in the address bar, and hit enter. If you don't know the IP address of your Ewon device, you can locate it using eBuddy.
+
+**Step 2:** Copy the files `EwonIgnitionDemoApp.jar` and `jvmrun` from the demo resource pack into the usr folder on your Ewon device using the FTP connection you've just opened.
+
+**Step 3:** Reboot your Ewon device by navigating to its web page, clicking the `Setup` tab, pressing `Reboot` and then pressing `REBOOT` once more.
+
+## Demo Dashboard Setup ##
+
+**Step 1:** Browse to your Ignition Gateway's web page using the URL from step 4 of Ignition Setup. Click the `Config` tab, and open `Projects`.
+
+**Step 2:** Click `Import Project`, select `Choose File`, and choose the file `EwonIgnitionDemo.zip` from the demo resource pack.
+
+**Step 3:** Assign the project a name of your choice, and click `Import`.
+
+**Step 4:** After successfully importing the Vision project, click the `Home` tab.
+
+**Step 5:** Download Vision Client Launcher by clicking `Download Vision Client Launcher`, and choosing your operating system.
+
+**Step 6:** After the download is complete, open the downloaded file, and follow the installation procedure to completion. The default settings are satisfactory, but you may modify them as desired.
+
+**Step 7:** After the installation is complete, open Vision Client Launcher if you have not done so already, and click `Add Vision Client`. For more information on location/opening Vision Client Launcher, see Starting the Demo step 1.
+
+**Step 8:** Double-click your Ignition gateway from the list of `On Your Network` gateways. Since your gateway is installed on the same computer as Ignition Vision Client Launcher, the URL field of your gateway should display `http://localhost:8088`. Please note that `8088` is a user-configurable port number, and your installation may not use port `8088` if you modified the port settings.
+
+**Step 9:** Ensure the Vision project you imported in Step 3 is selected, and click `Add Application(s)`. Please note, the imported Vision project may not use the name you chose earlier, and can be found as `HMS-Ewon-Ignition-Demo`
+
+**Step 10:** [Optional] Click the down arrow, located to the right of `Launch Fullscreen`, and click `Create Shortcut` to add a demo shortcut to your desktop for convenience.
+
+## Starting the Demo ##
+
+**Step 1:** Launch Vision Client Launcher by clicking it's icon in your operating system's main menu.
+
+* *Windows:* Start Button
+* *macOS:* Launchpad
+	* *Note: macOS Catalina (10.15) does not automatically add new applications to Launchpad. Open `Finder`, and click `Applications` to locate applications. To add an application to Launchpad, drag it to the Launchpad icon.*
```

Comment:
Do we really need to be this explicit in a readme?  If they cant use their OS thats really not our problem, especially when they are using an OS that is still in beta.

Reply:
I think its fine to just say launch Vision Client Launcher and leave it at that, but its fine to keep os specific instructions in.